torch/csrc/jit/python_compiled_function.cpp

```diff
@@ -200,18 +200,32 @@ CompiledFunction::TraceForKey* getTraceFor(CompiledFunction& fn,
 
 } // anonymous namespace
 
+static py::tuple tuple_tail(const py::tuple & tup) {
+  py::tuple r(tup.size() - 1);
+  for(int i = 1; i < tup.size(); i++) {
+    r[i-1] = tup[i];
+  }
+  return r;
+}
+
 void initCompilerMixin(PyObject *module) {
   auto m = py::handle(module).cast<py::module>();
   py::class_<CompiledFunction>(m, "CompiledFunction", py::dynamic_attr())
     .def(py::init<int, bool, bool, py::object, std::string>())
-    .def("__call__", [](CompiledFunction& fn, py::args args) -> py::object {
-      return fn.call(args);
+    .def("__call__", [](py::args args_) -> py::object {
+      auto fn = py::cast<CompiledFunction*>(args_[0]);
+      auto args = tuple_tail(args_);
+      return fn->call(args);
```

Comment:
ugh, what did happen in here?

Reply:
pybind11 leak. We can't fix it "properly" until upstream takes https://github.com/pybind/pybind11/pull/1216